aten/src/ATen/native/mkldnn/Utils.cpp

```diff
@@ -140,6 +140,13 @@ const std::map<c10::string_view, AttrFunction>& fx_fusion_attr_map() {
   return fusion_attr_map;
 };
 
+const std::map<c10::string_view, ideep::algorithm>& fusion_unary_alg_map() {
+  static const std::map<c10::string_view, ideep::algorithm> fusion_attr_map{
+      {"relu", {ideep::algorithm::eltwise_relu}},
+  };
+  return fusion_attr_map;
+};
+
```

Comment:
Why can't we reuse `fx_fusion_attr_map`? Also, should `fx_fusion_attr_map` be renamed to `fusion_unary_attr_map`?

Reply:
if we reuse **fx_fusion_attr_map**, we need to query the eltwise alg and its paraemters to re-create a new op attr, so I think we can have an alg map as binary and directly query the eltwise alg. I have changed the  **fx_fusion_attr_map** to **fusion_unary_attr_map**.